### PR TITLE
loading .dev.vars support for pages

### PR DESCRIPTION
**Pull request by Maximo-Guk**
#898

### CODE REVIEW

File: packages/wrangler/src/pages.tsx [hunk 1]
```diff
@@ -1417,6 +1417,16 @@ const createDeployment: CommandModule<
   },
 };
 
+function getDevVarsPath(): string {
```
Comment (threepointone) [inline packages/wrangler/src/pages.tsx:1420]:
```suggestion
function getDevVarsPath(): string | undefined {
```

File: packages/wrangler/src/pages.tsx [hunk 2]
```diff
@@ -1417,6 +1417,16 @@ const createDeployment: CommandModule<
...
+  const devVarsPath = "./.dev.vars";
+
+  if (existsSync(devVarsPath)) {
+    return devVarsPath;
+  }
+
+  return "";
```
Comment (threepointone) [inline packages/wrangler/src/pages.tsx:1427]:
```suggestion
```

File: packages/wrangler/src/pages/dev.tsx [hunk 3]
```diff
@@ -292,6 +299,7 @@ export const Handler = async ({
     kvPersist: true,
     durableObjectsPersist: true,
     cachePersist: true,
+    envPath: devVarsPath,
```
Comment (GregBrimble) [inline packages/wrangler/src/pages/dev.tsx:302]: I think we should use [straight bindings](https://miniflare.dev/core/variables-secrets/#bindings) like wrangler proper does: https://github.com/cloudflare/wrangler2/blob/57cf14e718749ab43f70034e21e600692cc90727/packages/wrangler/src/index.tsx#L1164

And ideally the `getVarsForDev` function that they've created as well.

This'll mean that Miniflare doesn't need to know anything about the fs, and will possibly make running wrangler in some other workdir easier in the future.

**Review by GregBrimble — APPROVED** [review]

**Review by GregBrimble — CHANGES_REQUESTED** [review]
Looks like you've accidentally included the config stuff too

File: .changeset/wise-pens-shake.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+"pages-functions-app": patch
```
Comment (GregBrimble) [inline .changeset/wise-pens-shake.md:2]:
```suggestion
```

**Review by GregBrimble — APPROVED** [review]
Great stuff!

**Pull request merged**